verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -282,11 +301,40 @@ function RowMenu(props: {
                         <span className={classes.menuItemTitle}>Edit</span>
                     </MenuItem>
                 </Link>
-                <MenuItem onClick={handleDelete}>
+                <MenuItem onClick={openDeleteDialog}>
                     <DeleteIcon />
                     <span className={classes.menuItemTitle}>Delete</span>
                 </MenuItem>
             </Menu>
+            <Dialog
+                open={deleteDialogOpen}
+                onClose={(): void => setDeleteDialogOpen(false)}
+                onClick={(e): void => e.stopPropagation()}
```

Comment:
is that for when you click on the dialog? what purpose does it serve?

Reply:
It stops the click being propagated to behind the dialog. Without this when you click on the dialog the onRowClick gets triggered and shows the ViewCase modal.